format-as-currency.js

```diff
@@ -58,10 +58,15 @@ angular
   return {
     require: 'ngModel',
     restrict: 'A',
+    scope: {
+      suppressCurrencySymbol: '@'
```

Comment:
- `@` is generally not good practice. consider observing the attribute (via `&`, `scope.$watch`, or `attrs.$observe`) instead, so that the app remains reactive, and changes in the currency symbol cascade down to this component.
- instead of simply suppressing the currency symbol, it's better to add the ability to pass in a more generic format function that you can use for anything (removing currency symbol, changing it, or changing `,` to `.`)
- it's better to avoid an isolate scope here, so the user can add more directives on the same element that has this directive on it. 1 scope per element is an angular limitation, and to keep this directive composable it's best to allow other directives to plug into the scope.